### ISSUE TITLE
Support disabling retries in config

### ISSUE THREAD

**Issue by daniel-deutsch-benchling** [svix/svix-webhooks#481]
### Motivation

One may want to disable retries altogether in webhook sending. For example, a use case where sending a webhook represents a stateful action, and even if `SVIX_WORKER_REQUEST_TIMEOUT` is hit, sending another request could have detrimental effects.

### Proposal

Pass an empty string to `SVIX_RETRY_SCHEDULE` should disable retries. This currently fails to parse at https://github.com/svix/svix-webhooks/blob/a697305f32fd5bdd06fc90ed1c25bcdf0daa1daf/server/svix-server/src/cfg.rs#L36

### Alternatives

Empty string feels most intuitive to me, but some separate boolean config to turn off retries also seems reasonable.

**Comment by tasn** [comment]
Thanks for the report, and I agree, an empty string feels most intuitive.

**Comment by tasn** [comment]
I also opened #480 earlier after looking at the config and seeing that we use a string instead of a toml array, I wonder whether it makes sense to fix this to accept arrays instead of strings (well, support strings for backwards compat), and then just pass the empty array (so `[]`) to indicate "empty".

**Comment by daniel-deutsch-benchling** [comment]
Yeah, I think #480 might be the way to go to simplify the string parsing.

I also realized after filing this that simply passing a single number without any commas like "1" representing a single retry attempt also does not parse correctly, although the error it hits is a bit further down:
```
webhooks_1   | thread 'main' panicked at 'Error loading configuration: Error { tag: Tag(Default, 3), profile: Some(Profile(Uncased { string: "default" })), metadata: Some(Metadata { name: "`SVIX_` environment variable(s)", source: None, provide_location: Some(Location { file: "svix-server/src/cfg.rs", line: 129, col: 10 }), interpolater:  }), path: ["retry_schedule"], kind: InvalidType(Unsigned(1), "a string"), prev: None }', svix-server/src/cfg.rs:131:10
```

Let me know if you want me to create a separate issue for this, or to leave as-is since it would also be fixed by #480.

**Comment by tasn** [comment]
Oh interesting, the "1" case fails when you set it in the env var, but not when you set it through the config. This actually seems like a bug in the upstream library we are using for parsing the environment.

Though yeah, I think this will also be fixed by #480.

**Comment by daniel-deutsch-benchling** [comment]
Thanks @tasn!
